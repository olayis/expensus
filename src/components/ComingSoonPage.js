import React from 'react';
import { Link } from 'react-router-dom';
import programming from '../../public/img/programming.svg';

const ComingSoonPage = () => (
  <main className='main-content'>
    <div className='content-container'>
      <div className='error'>
        <img
          src={programming}
          alt='Programmer'
          className='error__illustration'
          width='1075'
          height='585'
        />
        <h2 className='error__text'>This page is coming soon.</h2>
        <div className='error__explanation'>
          Continue managing your expenses.
          <span className='error__action'>
            <Link className='btn btn--try-again' to='/'>
              Go Home
            </Link>
          </span>
        </div>
      </div>
    </div>
  </main>
);

export default ComingSoonPage;
