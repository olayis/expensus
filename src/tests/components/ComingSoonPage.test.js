import React from 'react';
import { shallow } from 'enzyme';
import ComingSoonPage from '../../components/ComingSoonPage';

test('should render ComingSoonPage correctly', () => {
  const wrapper = shallow(<ComingSoonPage />);
  expect(wrapper).toMatchSnapshot();
});
