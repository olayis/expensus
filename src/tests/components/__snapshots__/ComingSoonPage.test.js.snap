// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ComingSoonPage correctly 1`] = `
<main
  className="main-content"
>
  <div
    className="content-container"
  >
    <div
      className="error"
    >
      <img
        alt="Programmer"
        className="error__illustration"
        height="585"
        src="SvgrURL"
        width="1075"
      />
      <h2
        className="error__text"
      >
        This page is coming soon.
      </h2>
      <div
        className="error__explanation"
      >
        Continue managing your expenses.
        <span
          className="error__action"
        >
          <Link
            className="btn btn--try-again"
            to="/"
          >
            Go Home
          </Link>
        </span>
      </div>
    </div>
  </div>
</main>
`;
